src/components/containers/show.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useContext, ReactNode } from 'react'
+import { Media } from 'themes/media'
+import { DerivStore } from 'store'
+
+type ResponseType = {
+    children?: ReactNode
+    max_width?: string & number
```

Comment:
we can use max_width:string | number right

Reply:
Unfortunately no.
We use dependency fresnel which should include `greaterThanOrEqual` which include string and number type in node_modules. I'm not yet realized how to change it
<img width="1031" alt="Screenshot 2022-07-28 at 15 00 41" src="https://user-images.githubusercontent.com/103182473/181504462-3503d986-92f4-4996-9916-32eeee1a3a5d.png">